README.md

```diff
@@ -222,7 +214,43 @@ This command does not touch repositories in which there are uncommitted changes.
 Examples:
 
 ```bash
-mgit update --recursive
+mgit update -- --recursive
+```
+
+### pull
+
+Pulls changes in existing repositories.
+
+If any dependency is missing, the command will install it too.
+
+Examples:
+
+```bash
+mgit pull -- --recursive
+```
+
+### push
+
+Pushes changes in existing repositories.
+
+If any dependency is missing, the command will not be executed.
```

Comment:
I don't understand it. If it doesn't exist then how can we be talking about pushing it?

Reply:
?

Reply:
Since this command pushes changes in existing repositories why do we need to say that it does nothing if a repository does not exist? If it doesn't exist how could it do anything? It can't. So it's a redundant information to me.